src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -409,7 +409,11 @@ public function testFormatWithDateTimeZoneGmtOffset()
 
     public function testFormatWithIntlTimeZone()
     {
-        if (PHP_VERSION_ID < 50500 && !(extension_loaded('intl') && method_exists('IntlDateFormatter', 'setTimeZone'))) {
+        if (!extension_loaded('intl')) {
+            $this->markTestSkipped('Int extension not installed.');
+        }
+
+        if (PHP_VERSION_ID < 50500 && !method_exists('IntlDateFormatter', 'setTimeZone')) {
```

Comment:
Aren't some (if not all) of those checks possible with an `@require` annotation now?

/cc @nicolas-grekas